Formula/s/sigstore.rb

```diff
@@ -189,21 +189,21 @@ def install
     assert_match version.to_s, shell_output("#{bin}/sigstore -V")
 
     resource "homebrew-test-artifact" do
-      url "https://github.com/sigstore/sigstore-python/releases/download/v3.3.0/sigstore-3.3.0.tar.gz", using: :nounzip
-      sha256 "931e9913996ceace713d28e2431989414e711af30606f0b1e8bdc30fcbdd3fbe"
+      url "https://github.com/sigstore/sigstore-python/releases/download/v3.4.0/sigstore-3.4.0.tar.gz", using: :nounzip
+      sha256 "71038afff4e0254aadc562cc725f6a1ccad7d32ff62abb3305c698737ee0a546"
     end
 
     resource "homebrew-test-artifact.sigstore" do
-      url "https://github.com/sigstore/sigstore-python/releases/download/v3.3.0/sigstore-3.3.0.tar.gz.sigstore"
-      sha256 "1cb946269f563b669183307b603f85169c7b1399835c66b8b4d28d913d26d5f7"
+      url "https://github.com/sigstore/sigstore-python/releases/download/v3.4.0/sigstore-3.4.0.tar.gz.sigstore"
+      sha256 "0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5"
     end
 
     resource("homebrew-test-artifact").stage testpath
     resource("homebrew-test-artifact.sigstore").stage testpath
 
-    cert_identity = "https://github.com/sigstore/sigstore-python/.github/workflows/release.yml@refs/tags/v3.3.0"
+    cert_identity = "https://github.com/sigstore/sigstore-python/.github/workflows/release.yml@refs/tags/v3.4.0"
 
-    output = shell_output("#{bin}/sigstore verify github sigstore-3.3.0.tar.gz --cert-identity #{cert_identity} 2>&1")
-    assert_match "OK: sigstore-3.3.0.tar.gz", output
+    output = shell_output("#{bin}/sigstore verify github sigstore-3.4.0.tar.gz --cert-identity #{cert_identity} 2>&1")
+    assert_match "OK: sigstore-3.4.0.tar.gz", output
```

Comment:
A couple of options here:

1. The test could keep using the old (v3.3.0) artifact -- it'll still verify correctly.
2. The test could use a CPython sigstore artifact instead, e.g. from 3.13: <https://www.python.org/downloads/release/python-3130/>
3. We could instead pull down a GitHub attestation and verify that (although that might be more annoying)

Reply:
(I'm happy to tweak this PR if any of those options sounds acceptable.)

Reply:
I think either 1 or 2 are fine, possibly 1 would be less work.

Reply:
Yep, 1 should be less work. I'll take over this PR and update it now.